docker/build_docker.sh

```diff
@@ -207,7 +207,23 @@ function docker_build {
         --label "org.opencontainers.image.authors=Demisto <containers@demisto.com>" \
         --label "org.opencontainers.image.version=${VERSION}" \
         --label "org.opencontainers.image.revision=${CIRCLE_SHA1}"
+
+    if [[ -e "dynamic_version.sh" ]]; then
+      echo "dynamic_version.sh file was found"
+      dynamic_version=$(docker run --rm -i "$image_full_name" sh < dynamic_version.sh)
+      echo "dynamic_version $dynamic_version"
```

Comment:
Let's log all the relevant variables
```suggestion
      echo "image_full_name: ${DOCKER_ORG}/${image_name}:$dynamic_version.${REVISION}"
```

Reply:
will add it the next line